.gitignore

```diff
@@ -1,3 +1,6 @@
+/.deps
+/.libs
+build/
```

Comment:
With "leave comments in the PR itself, not the commits" (paraphrased), do you mean this?

Reply:
yes, thank you! this way, when you open the PR you see all the comments.

Reply:
So, basically, you just go to the `Files changed` tab and you will see the complete list of changes. You don't need to go commit by commit. If you review commit by commit the comments go in that specific commit and its harder to find them, plus newer commits in the PR might be changing things on top of that, so better to look at the final list of changes.